tests/components/test_alexa.py

```diff
@@ -140,8 +146,41 @@ def test_intent_launch_request(alexa_client):
     }
     req = yield from _intent_req(alexa_client, data)
     assert req.status == 200
-    resp = yield from req.json()
-    assert "outputSpeech" in resp["response"]
+    data = yield from req.json()
+    text = data.get("response", {}).get("outputSpeech",
+                                        {}).get("text")
+    assert text == "LaunchRequest has been received."
+
+
+@asyncio.coroutine
+def test_intent_launch_request_not_configured(alexa_client):
+    """Test the launch of a request."""
+    data = {
+        "version": "1.0",
+        "session": {
+            "new": True,
+            "sessionId": SESSION_ID,
+            "application": {
+                "applicationId":
+                    'amzn1.echo-sdk-ams.app.000000-d0ed-0000-ad00-000000d00000'
+            },
+            "attributes": {},
+            "user": {
+                "userId": "amzn1.account.AM3B00000000000000000000000"
+            }
+        },
+        "request": {
+            "type": "LaunchRequest",
+            "requestId": REQUEST_ID,
+            "timestamp": "2015-05-13T12:34:56Z"
+        }
+    }
+    req = yield from _intent_req(alexa_client, data)
+    assert req.status == 200
+    data = yield from req.json()
+    text = data.get("response", {}).get("outputSpeech",
+                                        {}).get("text")
+    assert text == "This intent is not yet configured within Home Assistant."
```

Comment:
@balloob This line is a test case for what the response should be if you don't have an intent configured for the LaunchRequest.  There should be another test above it that tests for the proper response to a configured LaunchRequest skill.

Is this not correct?